pkg/generate/code/set_sdk.go

```diff
@@ -223,7 +223,7 @@ func SetSDK(
 			out += fmt.Sprintf(
 				"%s} else {\n", indent,
 			)
-			nameField := r.NameField()
+			nameField := *r.SpecIdentifierField()
```

Comment:
So I just noticed this... are you relying on the fact that the above will panic if the `SpecIdentifierField()` method returns `nil` for a resource?

Reply:
No, just wanted to double-check that this was your intention :) I'm totally fine with it